apps/linux/termite.talon

```diff
@@ -0,0 +1,15 @@
+os: linux
+app: /termite/
+not win.title: /VIM/
+-
+
+# Selection mode
+shell yank: key("y")
```

Comment:
i agree we can do that in general, although shell yank is slightly different, because is specifically related to visual mode in termite which is similar to vim visual mode. i'll fix the use of edit in general and commit in the next day or so hopefully

Reply:
i just checked and actually there's an apps/linux/terminal.talon that implements a bunch of shell-based actions that should be generic already, so will remove the ones that aren't termite-specific.